doc/Configuration.md

```diff
@@ -121,13 +124,17 @@ Supported values are:
 DESeq2 requires that your experiment design has at least one degree of freedom. If your experiment doesn't include at least one sample group with more than one replicate, DESeq2 will be skipped and DGE related plots will not be produced.
 
 ## Features Sheet Details
+| _Column:_ | Select for... | with value... | Alias by... | Tag | Hierarchy | Strand | 5' End Nucleotide | Length | Overlap | Feature Source |
+|--------------| --- | --- |-------------| --- | --- | --- | --- | --- |---------| --- |
+| _Example:_ | Class | CSR | Name        | tag1 | 1 | both | G | 22 | Exact   | ram1.gff3 |
```

Comment:
Let's change the example to something more common because CSR is specific to worms.   How about:  
Class | miRNA | Name |  | 1 | sense | all | all | 5' anchored | ram1.gff3